tokenserver/tests/test_memorynode.py

```diff
@@ -45,3 +45,15 @@ def test_assignation(self):
         user = self.backend.get_user(DEFAULT_EMAIL, DEFAULT_SERVICE)
         self.assertEquals(user['uid'], 1)
         self.assertEquals(user['node'], DEFAULT_NODE)
+
+    def test_AAA_assignment_spanner(self):
+        settings = self.config.get_settings()
+        settings['tokenserver.migrate_new_user_percentage'] = 1
+        self.config.add_settings(settings)
+
+        user0 = self.backend.allocate_user(
+            DEFAULT_SERVICE, 'user0@example.com')
+        user1 = self.backend.allocate_user(
+            DEFAULT_SERVICE, 'user1@example.com')
+        self.assertEquals(user0['node'], settings['tokenserver.spanner_entry'])
+        self.assertEquals(user1['node'], settings['tokenserver.service_entry'])
```

Comment:
This test feels like it could live in the more general `test_service.py` suite, rather than just being specific to the memory backend (which is the least important backend to test, since it's not the one we use in production).

I think it would also be worth adding a test to ensure the intended "stickiness" of this assignment, such as the user remaining assigned to spanner across a password change and password reset. Examples of similar tests can be found in `test_service.py`.